avaframe/in3Utils/fileHandlerUtils.py

```diff
@@ -665,10 +667,14 @@ def makeSimFromResDF(avaDir, comModule, inputDir='', simName=''):
 
     # Load input datasets from input directory
     if simName != '':
-        name = '*' + simName + '*.asc'
+        ascName = '*' + simName + '*.asc'
```

Comment:
see above

Reply:
Yes, we consider both, since we use the format of the dem